### PR TITLE
see_presentation

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
@strengejacke @pdwaggoner @mattansb any idea about how/what to showcase?

**Comment by mattansb** [comment]
Maybe something small - the new themes/geoms/`plots`/palettes?
And in any up-coming blog posts for the new releases we can insert the corresponding visualization from `see`? (with a teaser in this post of "more to come soon...")

**Comment by DominiqueMakowski** [comment]
I agree, since `see` is mainly a companion for the other packages/functions, it will be continuously showed along the way with their presentation. So yes, for this announcement post, maybe a few things specific to see, such as themes and geoms... We can basically add the first section of the README...

**Comment by strengejacke** [comment]
Yes, I agree. Once bayestestR is updated on CRAN, we may even combine see with a new bayestestR function?

**Comment by DominiqueMakowski** [comment]
Made 3 compact examples showcasing themes and some geoms. If you greenlight this I can merge tomorrow :)

**Comment by mattansb** [comment]
👍

**Comment by DominiqueMakowski** [comment]
@strengejacke @pdwaggoner

**Comment by strengejacke** [comment]
Just two small things... else, OK!

**Comment by DominiqueMakowski** [comment]
Let's go then. I hope the images will work on r-bloggers, but I see more and more posts without the images properly displaying. Apparently, the RSS parser of r-bloggers is too old and needs to be updated...

File: content/posts/see_presentation.Rmd [hunk 1]
```diff
@@ -21,36 +21,67 @@ include_graphics("https://github.com/easystats/see/raw/master/man/figures/logo.p
 
 
 
-## see
+## The see package
 
-[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life *easier*. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or  [visualisation](https://github.com/easystats/see).
+[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life easier. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or [visualisation](https://github.com/easystats/see).
```
Comment (pdwaggoner) [inline content/posts/see_presentation.Rmd:26]: Maybe: "...to make your statistics life easier." in the first sentence.

File: content/posts/see_presentation.Rmd [hunk 2]
```diff
@@ -21,36 +21,67 @@ include_graphics("https://github.com/easystats/see/raw/master/man/figures/logo.p
...
 
-***Without further ado, please let us introduce the latest addition to the easyverse; the ***  [**`see`**](https://github.com/easystats/see) ***package!***
+***Without further ado, please let us introduce the latest addition to the easyverse; the see ackage!***
```
Comment (pdwaggoner) [inline content/posts/see_presentation.Rmd]: Maybe: "***Without further ado, we introduce the latest addition to the easyverse: the see package!***"

A couple light grammatical things and streamlined language a bit. Feel free to keep or discard, of course

File: content/posts/see_presentation.Rmd [hunk 3]
```diff
@@ -21,36 +21,67 @@ include_graphics("https://github.com/easystats/see/raw/master/man/figures/logo.p
...
 
 
-<!-- The goal of [`performance`](https://github.com/easystats/performance) is to provide lightweight tools to **assess and check the quality of your model**. It includes functions such as [`R2`](https://easystats.github.io/performance/reference/r2.html) for many models (including logistic, mixed and Bayesian models), [`ICC`](https://easystats.github.io/performance/reference/icc.html) or helpers to check [`convergence`](https://easystats.github.io/performance/reference/check_convergence.html), [`overdipsersion`](https://easystats.github.io/performance/reference/check_overdispersion.html) or [`zero-inflation`](https://easystats.github.io/performance/reference/check_zeroinflation.html). See the **list of functions** [here](https://easystats.github.io/performance/reference/index.html). -->
+The goal of [`see`](https://github.com/easystats/see) is to be the visualisation companion to other functions and packages in **easystats**. See the **list of functions** [here](https://easystats.github.io/see/reference/index.html). However, it also includes some nice themes and geoms:
```
Comment (pdwaggoner) [inline content/posts/see_presentation.Rmd]: "...in the easyverse." 

Not sure if we want to start using "easystats" or "easyverse" more...?

**Comment by pdwaggoner** [comment]
Sorry for being late here - I just got back in town yesterday and am furiously catching up today. I had a few changes - some grammatical and some language-related. If it's in time, feel free to review and accept (or not). Otherwise, everything looks great here!

**Pull request merged**